h/views/account_signup.py

```diff
@@ -13,12 +13,8 @@
 _ = i18n.TranslationString
 
 
-def _login_redirect_url(request):
-    return request.route_url("activity.user_search", username=request.user.username)
```

Comment:
This helper function was only called once. Inlined it into the function below that calls it.